### PR TITLE
Convince your boss, press pack, etc

### PR DESCRIPTION
**Pull request by plaindocs**
This is a franken monster of cobbled letters, tweets and other goodness.

### CODE REVIEW

**Comment by plaindocs** [comment]
@kelly-o got thoughts? feelings? words?

File: docs/conf/portland/2018/press-kit.rst [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+  TODO
+
+  TODO
+
+  Sincerely,
+  [your name]
+
+Images and Logos
+-------------------
```
Comment (ericholscher) [inline docs/conf/portland/2018/press-kit.rst]: Should at least point to the resources repo.

Comment (plaindocs) [reply docs/conf/portland/2018/press-kit.rst]: Well yeah, still have a number of todos to do :-D

**Comment by ericholscher** [comment]
Would love to get this moved along, and at least have something for a "press pack".

**Comment by plaindocs** [comment]
Aye. I'll get on the case.

File: docs/conf/portland/2018/press-kit.rst [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+---------------
+
+We have some stats about attendees, membership, and website hits over on the following blog posts:
+
+- :doc:`/blog/write-the-docs-2017-stats/`
+- :doc:`/blog/write-the-docs-2016-year-in-review/`
+
+
+Images and Logos
+-------------------
```
Comment (ericholscher) [inline docs/conf/portland/2018/press-kit.rst]: I'd love to put in some group photos and other useful "use this to talk about us" photos. A lot of people post crappy photos in articles, so it would be cool to have a set we recommend.

File: docs/conf/portland/2018/press-kit.rst [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+
+  I'd like to attend Write the Docs in Portland, 6-8 May 2018. A three-day event that explores the art and science of documentation. The conference covers topics related to documentation in the software industry.
+
+  Write the Docs brings everyone who writes the docs together in the same room: Writers, Developers, Developer Relations, and Support Staff, which makes it an ideal networking opportunity .
+
+  There is a single talk track, a parallel unconference event and a writing day. The `sessions from 2017 <http://www.writethedocs.org/conf/na/2017/speakers/>`_ provide a reasonable idea of what to expect.
+
+  Cost:
+
+  * ticket $450
```
Comment (ericholscher) [inline docs/conf/portland/2018/press-kit.rst]: Feels a bit informal with the lower casing. Could make it a bit more "employer ready" I think.

**Review by ericholscher — COMMENTED** [review]
Looks like a good start. I'll take a swing at adding some more stuff over time, but we should merge it sooner rather than later.

Also need to link to it from the nav on the actual site to make it "live".

File: docs/conf/portland/2018/press-kit.rst (hunk 2 above)
Comment (plaindocs) [reply docs/conf/portland/2018/press-kit.rst]: I've asked @goatlady to pick us out 3 of her faves, including a group shot and a venue shot.

Comment (plaindocs) [reply docs/conf/portland/2018/press-kit.rst]: What do we think of these: 

1: https://www.flickr.com/photos/writethedocs/34495135662/in/album-72157683817839465/
2: https://www.flickr.com/photos/writethedocs/33849416753/in/album-72157683817839465/
3: https://www.flickr.com/photos/writethedocs/33856825394/in/album-72157683817839465/

Comment (ericholscher) [reply docs/conf/portland/2018/press-kit.rst]: 👍

File: docs/conf/portland/2018/press-kit.rst [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
+
+Sample Emails
+--------------
+
+Convince Your Employer
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Do you need help justifying why your employer should send you to the Write the Docs? Take a look at the letter below and tweak it as required.
+
+  FROM: [your name]
```
Comment (thatdocslady) [inline docs/conf/portland/2018/press-kit.rst]: Do we want to do that thing where we embed the email text as plain text, like we do in the email templates?

File: docs/conf/portland/2018/press-kit.rst [hunk 5]
```diff
@@ -0,0 +1,95 @@
...
+    <li>Join writers, programmers, librarians, scientists, developer advocates. and support humans at #writethedocs in Portland, 6-8 May 2018 to discuss all things documentation. <br><a href="https://twitter.com/share?ref_src=twsrc%5Etfw" class="twitter-share-button" data-text="Join writers, programmers, librarians, scientists, developer advocates. and support humans at #writethedocs in Portland, 6-8 May 2018 to discuss all things documentation." data-url="http://www.writethedocs.org/conf/portland/2018/" data-hashtags="writethedocs" data-show-count="false">Tweet</a><script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script></li>
+  </ul>
+
+Sample Emails
+--------------
+
+Convince Your Employer
+~~~~~~~~~~~~~~~~~~~~~~~~~~
```
Comment (thatdocslady) [inline docs/conf/portland/2018/press-kit.rst]: delete unnecessary ~~~~~

File: docs/conf/portland/2018/press-kit.rst [hunk 6]
```diff
@@ -0,0 +1,95 @@
+:template: 2018/na/generic.html
+
+Press Kit
+=========
+
+We love that you're supporting Write the Docs by writing about the conference, sharing news about it on social media, or asking your manager if you can attend. We're trying to make it as easy as possible for you to do any of the above, so here are some sample tweets, emails, and images you can use.
+
+Sample Tweets
```
Comment (thatdocslady) [inline docs/conf/portland/2018/press-kit.rst]: Sentence-case for H2 and below (global)

File: docs/conf/portland/2018/press-kit.rst [hunk 7]
```diff
@@ -0,0 +1,95 @@
...
+    <li>Join writers, programmers, librarians, scientists, developer advocates. and support humans at #writethedocs in Portland, 6-8 May 2018 to discuss all things documentation. <br><a href="https://twitter.com/share?ref_src=twsrc%5Etfw" class="twitter-share-button" data-text="Join writers, programmers, librarians, scientists, developer advocates. and support humans at #writethedocs in Portland, 6-8 May 2018 to discuss all things documentation." data-url="http://www.writethedocs.org/conf/portland/2018/" data-hashtags="writethedocs" data-show-count="false">Tweet</a><script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script></li>
+  </ul>
+
+Sample Emails
+--------------
+
+Convince Your Employer
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Do you need help justifying why your employer should send you to the Write the Docs? Take a look at the letter below and tweak it as required.
```
Comment (thatdocslady) [inline docs/conf/portland/2018/press-kit.rst]: "to the Write the Docs" <conference? event?> (needs a qualifier)

File: docs/conf/portland/2018/press-kit.rst [hunk 8]
```diff
@@ -0,0 +1,95 @@
...
+
+  FROM: [your name]
+  TO: [your employer or manager's name]
+  RE: Continuous Improvement and Education
+
+  I'd like to attend Write the Docs in Portland, 6-8 May 2018. A three-day event that explores the art and science of documentation. The conference covers topics related to documentation in the software industry.
```
Comment (thatdocslady) [inline docs/conf/portland/2018/press-kit.rst]: "It's a 3-day conference that..."

File: docs/conf/portland/2018/press-kit.rst [hunk 9]
```diff
@@ -0,0 +1,95 @@
...
+
+  Write the Docs brings everyone who writes the docs together in the same room: Writers, Developers, Developer Relations, and Support Staff, which makes it an ideal networking opportunity .
+
+  There is a single talk track, a parallel unconference event and a writing day. The `sessions from 2017 <http://www.writethedocs.org/conf/na/2017/speakers/>`_ provide a reasonable idea of what to expect.
```
Comment (thatdocslady) [inline docs/conf/portland/2018/press-kit.rst]: "provide a reasonable idea" doesn't sound very exciting, maybe rephrase to something like "show a diverse range of topics that are relevant for my work"

File: docs/conf/portland/2018/press-kit.rst [hunk 10]
```diff
@@ -0,0 +1,95 @@
...
+
+  Cost:
+
+  * ticket $450
+  * travel
+  * accomodation
+
+  Benefits:
+
+  * networking with other documentarians
```
Comment (thatdocslady) [inline docs/conf/portland/2018/press-kit.rst]: maybe explain documenarian here (i.e. folks who are passionate about documentation from all roles in the tech fields)

File: docs/conf/portland/2018/press-kit.rst [hunk 11]
```diff
@@ -0,0 +1,95 @@
...
+  * exposure to new or state of the art ideas/techniques/products
+  * a chance to contribute to discussions and working groups
+  * find solutions to problems you're facing at work
+
+  Sincerely,
```
Comment (thatdocslady) [inline docs/conf/portland/2018/press-kit.rst]: Thanks in advance (sounds more like "ahem ahem" ;) )

File: docs/conf/portland/2018/press-kit.rst [hunk 12]
```diff
@@ -0,0 +1,95 @@
...
+  * ticket $450
+  * travel
+  * accomodation
+
+  Benefits:
+
+  * networking with other documentarians
+  * exposure to new or state of the art ideas/techniques/products
+  * a chance to contribute to discussions and working groups
+  * find solutions to problems you're facing at work
```
Comment (thatdocslady) [inline docs/conf/portland/2018/press-kit.rst]: you're > I'm (this is a letter template so we should assume people will copy/paste and send under their name)

File: docs/conf/portland/2018/press-kit.rst (hunk 10 above)
Comment (thatdocslady) [reply docs/conf/portland/2018/press-kit.rst]: also, maybe add something about potential hiring connections?

File: docs/conf/portland/2018/press-kit.rst [hunk 13]
```diff
@@ -0,0 +1,95 @@
...
+  Cost:
+
+  * ticket $450
+  * travel
+  * accomodation
+
+  Benefits:
+
+  * networking with other documentarians
+  * exposure to new or state of the art ideas/techniques/products
```
Comment (thatdocslady) [inline docs/conf/portland/2018/press-kit.rst]: state-of-the-art (isn't is usually hyphenated?)

**Review by thatdocslady — COMMENTED** [review]
Looks great, bunch of nits :)

File: docs/conf/portland/2018/press-kit.rst (hunk 6 above)
Comment (kelly-o) [reply docs/conf/portland/2018/press-kit.rst]: 👍

File: docs/conf/portland/2018/press-kit.rst (hunk 5 above)
Comment (kelly-o) [reply docs/conf/portland/2018/press-kit.rst]: 👍

File: docs/conf/portland/2018/press-kit.rst (hunk 7 above)
Comment (kelly-o) [reply docs/conf/portland/2018/press-kit.rst]: just deleted the 'the'

File: docs/conf/portland/2018/press-kit.rst (hunk 8 above)
Comment (kelly-o) [reply docs/conf/portland/2018/press-kit.rst]: 👍

File: docs/conf/portland/2018/press-kit.rst (hunk 9 above)
Comment (kelly-o) [reply docs/conf/portland/2018/press-kit.rst]: 👍

File: docs/conf/portland/2018/press-kit.rst (hunk 3 above)
Comment (kelly-o) [reply docs/conf/portland/2018/press-kit.rst]: 👍

File: docs/conf/portland/2018/press-kit.rst (hunk 13 above)
Comment (kelly-o) [reply docs/conf/portland/2018/press-kit.rst]: 👍 (yep, when it's a compound modifier like this)

File: docs/conf/portland/2018/press-kit.rst (hunk 12 above)
Comment (kelly-o) [reply docs/conf/portland/2018/press-kit.rst]: 👍

File: docs/conf/portland/2018/press-kit.rst (hunk 11 above)
Comment (kelly-o) [reply docs/conf/portland/2018/press-kit.rst]: 👍

**Review by kelly-o — COMMENTED** [review]
just going through and incorporating everybody's copy edits first

**Comment by plaindocs** [comment]
Gah, doesn't handle multi speakers. I deleted too much stuff. Brb

**Comment by plaindocs** [comment]
OK, as far as @kelly-o and I are concerned this is READY TO ROCK!

@thatdocslady wanna have a look see or shall we just push it out?

**Comment by thatdocslady** [comment]
Ship it!

**Comment by plaindocs** [comment]
Annoyed that sphinx didn't warn about broken links either.

http://www.sphinx-doc.org/en/stable/config.html#confval-nitpicky doesn't seem to help :-(

Might have to look into htmlproofer for local links only

**Comment by ericholscher** [comment]
Are we running a linkchecker? I imagine we have _a lot_ of broken links. If we fix them, we could add it to our travis.

**Comment by plaindocs** [comment]
We're not because we can't cos we have *so* many broken links. 

There is an issue #230 or three about this. I expected sphinx to catch this cos internal. 

Andrea and I are talking about making time to fix them as well.
